File: POS_line.py
# coding=UTF-8
import json
import sys
import requests

filename = sys.argv[1]
outputdir = sys.argv[2]

rawData = ''

with open(sys.argv[1], encoding='utf8') as source:
    rawData = json.load(source)

results = []

for (index, line) in enumerate(rawData):
    print(index, line['lineId'])
    res = json.loads(requests.post('http://localhost:8999', data=line['sentences'].encode('utf-8')).text)
    result = {
        'lineId': line['lineId'],
        'roleId': line['roleId'],
        'movieId': line['movieId'],
        'roleName': line['roleName'],
        'sentences': line['sentences'],
        'parsing': res
    }
    results.append(result)
    if len(results) % 100 == 0:
        with open(outputdir + '/output_' + str(int(index / 100) + 1) + '.json', 'w', encoding='utf-8') as output:
            json.dump(results, output)
        results = []

# remaining
with open(outputdir + '/output_' + str(int(index / 100) + 1) + '.json', 'w', encoding='utf-8') as output:
    json.dump(results, output)

File: POS_line.js
var fs = require('fs-extra');
var rp = require('request-promise');

let inputFilename = process.argv[2]
let outputFilename = process.argv[3] || 'output.json'

let input = fs.readFileSync(inputFilename, 'utf8')

let lines = input.split(/\n/).map((line) => {
    let split = line.split(' +++$+++ ')
    return {
        lineId: split[0],
        roleId: split[1],
        moiveId: split[2],
        roleName: split[3],
        sentences: split[4]
    }
})

function promiseGetPOS(raw) {
    return rp({
        method: 'POST',
        uri: 'http://localhost:8999',
        body: raw.sentences
    }).then((res) => {
        return {
            raw: raw,
            result: JSON.parse(res)
        };
    });
}

var lineChunks = []

while(lines.length) {
    lineChunks.push(lines.splice(0,1000))
}

lineChunks.forEach((lineChunk, index) => {
    let tasks = lineChunk.map((element) => promiseGetPOS(element));

    Promise.all(tasks)
        .then((result) => {
            console.log('part' + index + ' - ' + (index * 1000) + '~' + ((index + 1) * 1000 - 1))
            fs.writeJsonSync('part' + index + '_' + outputFilename, result)
            console.log(result)
        })
        .catch((err) => {
            console.log('part' + index + ' - ' + (index * 1000) + '~' + ((index + 1) * 1000 - 1))
            console.error(err)
        })
})